_posts/2016-10-23-recurse-center-week-4.md

```diff
@@ -0,0 +1,158 @@
+
+I disconnected, went into a room and started working on a PR for `nodepub`. It's an open source library for bundling HTML into `epub` files.
+
+The library has been working well for me, but unfortunately was using a some synchronous apis. In the NodeJS world, this means the entire server can get blocked on writing epub files. I thought making the code fully asynchronous would be a fun little project. The last two days of the week I did that and opened a PR.
+
+#### **Support from RCers**
+
+You know what we all need to do more of? CODE REVIEWS.
+
+I've been trying to get into the habit of opening PRs against my own repos for RCers to review. Did that with a bunch of my blog posts and was super appreciative to get RC perspective on them.
+
+Now I just need to do that with...code. 😜
+
+#### **Non-technical lightning talks**
+
+The people at RC come from such a wide background, and sometimes you forget that when all you talk about is coding.
+
+Non-technical lightning talks was a chance for everyone to share some of their non-technical interests. It was so interesting hearing people talk on everything from Disco to Judo to Brexit. I gave a talk on Nonviolent Communication (NVC) which was a lot of fun 😊.
+
+#### **Finished "Algorithms to Live By"**
+
+I finished reading [this amazing book](https://www.goodreads.com/book/show/25666050-algorithms-to-live-by) called "Algorithms to Live By"!
+
+It was a super interesting mixture of computer science concepts and life advice. Reviewing old CS concepts and seeing how the author related those concepts to real life was really interesting. Would recommend 👍.
+
+#### **I went to BrooklynJS**
+
+A friend at RC was speaking at the BrooklynJS meet up on Thursday and provided me with his guest ticket. Lovely crowd, lovely talks.
+
+One of the highlights was meeting [Brian Brennan](https://github.com/brianloveswords). I watched a talk they gave at JS Conf a while back called ["Being Human"](https://www.youtube.com/watch?v=LlO2_GecWo8). It was such a wonderfully presented overview of many thoughts I had been pondering.
+
+So happy to find myself at a meetup mentioned in the talk.
+
+#### **I started going to interview prep**
+
+Fridays at RC are quieter than usual. It's a time when people focus more on interview prep rather then coding.
+
+I'm not decided on my post-rc plans, so interview prep hadn't appealed to me. At the same time, it's a skill that is harder to practice solo. So last week I joined a session and collaborated on problems. Made me realize how much I could improve on my whiteboard skills. So happy I did it.
+
+Will sign up for more over the coming weeks.
+
+#### **I'm out of touch with my technical goals**
+
+All this running around, socializing and collaborating has really pulled me away from deep diving into my own personal tech problem.
+
+I might need to take some time to focus on Harold. What do I want to learn? Why? Am I challenging myself?
```

Comment:
This is definitely true. I'll need to add this time to my calendar. So much easier to just say it though :P.